docs/iceberg-rest-service.md

```diff
@@ -130,14 +130,18 @@ To configure the JDBC catalog backend, set the `gravitino.iceberg-rest.warehouse
-|------------------------------------------------|-------------------------------------------------------------------------------------------------------|---------------|----------|------------------|
-| `gravitino.iceberg-rest.io-impl`               | The IO implementation for `FileIO` in Iceberg, use `org.apache.iceberg.aliyun.oss.OSSFileIO` for OSS. | (none)        | No       | 0.6.0-incubating |
-| `gravitino.iceberg-rest.oss-access-key-id`     | The static access key ID used to access OSS data.                                                     | (none)        | No       | 0.7.0-incubating |
-| `gravitino.iceberg-rest.oss-secret-access-key` | The static secret access key used to access OSS data.                                                 | (none)        | No       | 0.7.0-incubating |
-| `gravitino.iceberg-rest.oss-endpoint`          | The endpoint of Aliyun OSS service.                                                                   | (none)        | No       | 0.7.0-incubating |
-
-For other Iceberg OSS properties not managed by Gravitino like `client.security-token`, you could config it directly by `gravitino.iceberg-rest.client.security-token`.
+| Configuration item                                | Description                                                                                            | Default value   | Required | Since Version    |
+|---------------------------------------------------|--------------------------------------------------------------------------------------------------------|-----------------|----------|------------------|
+| `gravitino.iceberg-rest.io-impl`                  | The IO implementation for `FileIO` in Iceberg, use `org.apache.iceberg.aliyun.oss.OSSFileIO` for OSS.  | (none)          | No       | 0.6.0-incubating |
+| `gravitino.iceberg-rest.oss-access-key-id`        | The static access key ID used to access OSS data.                                                      | (none)          | No       | 0.7.0-incubating |
+| `gravitino.iceberg-rest.oss-secret-access-key`    | The static secret access key used to access OSS data.                                                  | (none)          | No       | 0.7.0-incubating |
+| `gravitino.iceberg-rest.oss-endpoint`             | The endpoint of Aliyun OSS service.                                                                    | (none)          | No       | 0.7.0-incubating |
+| `gravitino.iceberg-rest.oss-region`               | The region of the OSS service, like `oss-cn-hangzhou`.                                                 | (none)          | No       | 0.8.0-incubating |
+| `gravitino.iceberg-rest.oss-role-arn`             | The ARN of the role to access the OSS data.                                                            | (none)          | No       | 0.8.0-incubating |
```

Comment:
could you add only used when `credential-provider-type` is `oss-token`?

Reply:
fixed